store/builder.go

```diff
@@ -3,26 +3,29 @@ package store
 import (
 	"context"
 	"fmt"
+	"math/rand"
 	"time"
 
 	"github.com/Layr-Labs/eigenda-proxy/common"
 	"github.com/Layr-Labs/eigenda-proxy/metrics"
 	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/eigenda"
 	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore"
 	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore/memconfig"
-	memstorev2 "github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore/v2"
-	eigendav2 "github.com/Layr-Labs/eigenda-proxy/store/generated_key/v2"
+	memstore_v2 "github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore/v2"
+	eigenda_v2 "github.com/Layr-Labs/eigenda-proxy/store/generated_key/v2"
 	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/redis"
 	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/s3"
 	"github.com/Layr-Labs/eigenda-proxy/verify/v1"
 	"github.com/Layr-Labs/eigenda/api/clients"
 	clients_v2 "github.com/Layr-Labs/eigenda/api/clients/v2"
 	"github.com/Layr-Labs/eigenda/api/clients/v2/verification"
-
+	common_da "github.com/Layr-Labs/eigenda/common"
```

Comment:
nit - we use `common_eigenda` in other parts of the repo

Reply:
[5556968](https://github.com/Layr-Labs/eigenda-proxy/pull/317/commits/555696823aaf0b6b73b2e3ebf8cc88444210bf4a)